DistTasks.yml

```
@@ -251,3 +251,9 @@ tasks:
       CONTAINER_TAG: "{{.GO_VERSION}}-darwin-arm64-debian10"
       PACKAGE_PLATFORM: "macOS_ARM64"
       PACKAGE_NAME: "{{.PROJECT_NAME}}_{{.VERSION}}_{{.PACKAGE_PLATFORM}}.tar.gz"
+
+  dist:jsonschema:
```

```suggestion
  jsonschema:
```
THe CI is failing because the `dist`is already added [here](https://github.com/arduino/arduino-cli/blob/b6cc517d68874795b60a0a4bfe04b068bf8540ee/Taskfile.yml#L4)